wharf-helm/templates/api.yaml

```
@@ -0,0 +1,131 @@
+spec:
+  replicas: {{ .Values.api.replicaCount }}
+  selector:
+    matchLabels:
+      component: api
+      {{- include "wharf-helm.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        component: api
+        {{- include "wharf-helm.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets: {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "wharf-helm.serviceAccountName" . }}
+      {{- with .Values.api.podSecurityContext }}
+      securityContext: {{- toYaml . | nindent 12 }}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name | quote }}
+          {{- with .Values.api.containerSecurityContext }}
+          securityContext: {{- toYaml . | nindent 12 }}
+          {{- end }}
+          image: {{ .Values.api.image | quote }}
+          {{- with .Values.api.imagePullPolicy }}
+          imagePullPolicy: {{ . | quote }}
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.api.containerPort }}
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: http
+          env:
+            {{- with .Values.global.instanceId }}
+            - name: WHARF_INSTANCE
+              value: {{ quote . }}
+            {{- end }}
+            - name: CI_URL
+              {{- .Values.api.ciUrl | include "wharf-helm.environmentValue" | nindent 14 }}
+            - name: CI_TOKEN
+              {{- .Values.api.ciToken | include "wharf-helm.environmentValue" | nindent 14 }}
+
+            {{- with .Values.api.rabbitmq }}
+            {{- if .enabled }}
+            - name: RABBITMQENABLED
+              value: "true"
+            - name: RABBITMQUSER
+              {{- .username | include "wharf-helm.environmentValue" | nindent 14 }}
+            - name: RABBITMQPASS
+              {{- .password | include "wharf-helm.environmentValue" | nindent 14 }}
+            - name: RABBITMQHOST
+              {{- .host | include "wharf-helm.environmentValue" | nindent 14 }}
+            - name: RABBITMQPORT
+              {{- .port | include "wharf-helm.environmentValue" | nindent 14 }}
+            - name: RABBITMQVHOST
+              {{- .vHost | include "wharf-helm.environmentValue" | nindent 14 }}
+            - name: RABBITMQNAME
+              {{- .name | include "wharf-helm.environmentValue" | nindent 14 }}
+            - name: RABBITMQCONNATTEMPTS
+              {{- .connAttempts | include "wharf-helm.environmentValue" | nindent 14 }}
+            {{- else }}
+            - name: RABBITMQENABLED
+              value: "false"
+            {{- end }}{{/* if .enabled */}}
+            {{- end }}{{/* with .Values.api.rabbitmq */}}
+
+						{{- with .Values.api.db }}
```

There is a strange space here.
```suggestion
            {{- with .Values.api.db }}
```

---

Ah yes, my editor must've switched to tabs instead of spaces for just that indent.